client/src/pages/profile/index.tsx

```diff
@@ -80,375 +43,185 @@ const Profile = (): ReactElement => {
-   *
-   * @param field the field of the user to edit
-   * @param value the value corresponding to that field
-   */
-  const editUser = <T extends keyof IUser>(field: T, value: IUser[T]): void =>
-    setUser(updateUserField(user, field, value));
-
-  /**
-   * Gets the cumulative options for a dropdown including
-   * user's current options
-   *
-   * @param dropdown the dropdown to get
-   * @returns the parsed semantic UI dropdown options
-   */
-  const getMultiDropdownOptions = (
-    dropdown: MultiDropdowns,
-  ): DropdownItemProps[] =>
-    parseOptions(dropdownOptions[dropdown].concat(user[dropdown]));
-
-  /**
-   * Adds an option to a dropdown
-   *
-   * @param dropdown the dropdown to add the option to
-   * @param value the value to add
-   */
-  const addDropdownOption = (dropdown: MultiDropdowns, value: string): void => {
-    let allOptions;
-
-    if (dropdown === 'genders') {
-      allOptions = allGenders;
-    } else if (dropdown === 'pronouns') {
-      allOptions = allPronouns;
-    } else {
-      allOptions = allRaces;
-    }
-
-    updateDropdownOptions(dropdown, value, allOptions);
-  };
-
   return (
-    <div>
+    <div className="profile-page">
       <Grid padded stackable>
-        <Grid.Row columns={4}>
-          <Grid.Column className="profile-pic-col">
-            {(userId === auth.user._id || auth.isAdmin) && (
-              <Button
-                size="medium"
-                className="edit-profile-btn"
-                onClick={startEditMode}
-                content="Edit Profile"
-              />
-            )}
-
-            <UserPicture size="small" user={user} />
+        <Grid.Row columns={5}>
+          <Grid.Column className="profile-pic-col" text-align="left" width={4}>
+            <div className="user-pic">
+              {' '}
+              <UserPicture size="tiny" user={user} />
+            </div>
+            <div className="name-pronouns">
+              {user.preferredName !== '' && user.preferredName ? (
+                <h2 className="name">{`${titleCase(
+                  user.preferredName,
+                )} (${titleCase(getUserFullName(user))})`}</h2>
+              ) : (
+                <h2>{titleCase(getUserFullName(user))}</h2>
+              )}
+              {user.pronouns.map((pronoun, index) => (
+                <FieldTag
+                  key={index}
+                  name={pronoun.toLowerCase()}
+                  content="pronouns"
+                />
+              ))}
+              <div className="user-role">
+                <FieldTag content={user.role} />
+              </div>
+              <div className="rating">
+                {/* TODO: update when rating is added to model*/}
+                <Rating icon="star" defaultRating={3} maxRating={5} />
+                <p className="number-ratings">(16)</p>
+              </div>
+              <div>
+                {(userId === auth.user._id || auth.isAdmin) && (
```

Comment:
This is the whole point of that isEditable thing and so only certain people could see certain information on the page. If you removed all of that, seems like some of the functionality of this page probably got deleted too.

Reply:
I had removed isEditable because I thought this would be abstracted into the edit profile modal. However, you're right I should keep isViewable for permissions on what the user can view.